Sources/Apollo/MultipartResponseParsingInterceptor.swift

```diff
@@ -112,6 +112,7 @@ public struct MultipartResponseParsingInterceptor: ApolloInterceptor {
             )
 
             // These are fatal-level transport errors, don't process anything else.
+            chain.terminate()
```

Comment:
Do we also need a `terminate` on line 99 here?

Reply:
No because there could be more chunks arriving after all other errors. It's only fatal-level transport errors that we can assume the router/server will close the connection.